code/game/machinery/teleporter.dm

```diff
@@ -69,13 +69,13 @@
 		if(do_teleport(M, com.target, channel = TELEPORT_CHANNEL_BLUESPACE))
 			use_power(5000)
 
-			if(!calibrated && iscarbon(M) && prob(30 - ((accurate) * 10))) //oh dear a problem
+			if(!calibrated && iscarbon(M) && prob(0 - ((accurate) * 0))) //oh dear a problem
```

Comment:
I would rather this section be commented out, and noted with `KEPLER CHANGE: Removes flyperson chance`, as setting the prob to 0 is a very hacky way of doing this, and is not ideal in the slightest.